samples/read_fast.c

```diff
@@ -94,6 +96,7 @@ int main(int argc, char *argv[])
             }
```

Comment:
This is an old issue, but I just spotted it in testing.  We need to add +33 to avoid printing up rubbish to the terminal.

```
diff --git a/samples/read_fast.c b/samples/read_fast.c
index 3ed6ba1f..642b1a19 100644
--- a/samples/read_fast.c
+++ b/samples/read_fast.c
@@ -92,7 +92,7 @@ int main(int argc, char *argv[])
         if (infile->format.format == fastq_format) {
             printf("\nquality: ");
             for (c = 0; c < bamdata->core.l_qseq; ++c) {
-                printf("%c", bam_get_qual(bamdata)[c]);
+                printf("%c", bam_get_qual(bamdata)[c]+33);
             }
         }
     }
```

Reply:
updated.